synapse/lib/jsonstor.py

```diff
@@ -57,12 +57,13 @@ def _incRefObj(self, buid, valu=1):
         refs += valu
         if refs > 0:
             self.slab.put(buid + b'refs', s_msgpack.en(refs), db=self.metadb)
+            await asyncio.sleep(0)
```

Comment:
The extra sleeps probably don't hurt anything, but IMO we only need the sleep in the loop removing the meta entries here which could potentially be hot, don't really need this sleep and the other two after single `put` calls.